### PR TITLE
Implement changes for fourth day

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Description
- Create vets table
- Create a specializations table
- Create a visits table
- Insert data into the newly created tables
- Write Queries for the insertions

## These are the pictures of the queries
Screenshot 2023-05-13 110636
Screenshot 2023-05-13 110700

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -18,4 +18,41 @@ SELECT * FROM animals;
...
- SELECT * FROM animals;
\ No newline at end of file
+ SELECT * FROM animals;
+
+   insert into vets(name, age, date_of_graduation)
+  values
+    ('William Tatcher', 45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-02-11');
```
Comment (binhussen) [inline data.sql]: - Kindly, insert the correct data `Jack Harkness` birth date is `2008-06-08` so update it to follow the requirement 👍

File: schema.sql [hunk 2]
```diff
@@ -10,4 +10,78 @@
...
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES
+  ('Pokemon'),
+  ('Digimon');
```
Comment (binhussen) [inline schema.sql]: - Kindly, Everything that populates tables with data should be saved in the data.sql file as you are inserting data here so move it to data.sql to put the correct SQL commands in correct place👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Well Structured Query✔️
- Descriptive pull request with screenshots✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @IsmailChangezi , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**